pennylane/transforms/qcut/cutcircuit.py

```diff
@@ -46,7 +46,20 @@ def _cut_circuit_expand(
     def processing_fn(res):
         return res[0]
 
-    return [_qcut_expand_fn(tape, max_depth, auto_cutter)], processing_fn
+    tapes, tapes_fn = [tape], processing_fn
```

Comment:
Should we move this line below now?

Reply:
I don't think it makes any difference. 🤔